### PR TITLE
[ARTS-255] support VS Code devcontainer 

### PR DESCRIPTION
**Pull request by balteravishay**
This PR adds support for VS code [devcontainers](https://code.visualstudio.com/docs/remote/containers).

### CODE REVIEW

**Review by tamirkamara — APPROVED** [review]

**Review by eladiw — APPROVED** [review]

**Review by LizaShak — APPROVED** [review]

File: .devcontainer/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.154.0/containers/java/.devcontainer/base.Dockerfile
+
+# [Choice] Java version: 11, 15
+ARG VARIANT="15"
```
Comment (itye-msft) [inline .devcontainer/Dockerfile:4]: Why is 15 an option? I think we are going only with 11, don't we?

**Review by itye-msft — APPROVED** [review]
Please see my comment

File: .devcontainer/Dockerfile (hunk 1 above)
Comment (shajidissa) [reply .devcontainer/Dockerfile:4]: +1

**Review by shajidissa — APPROVED** [review]
All ok

**Pull request merged**